api/src/test/java/org/pmiops/workbench/rdr/RdrExportServiceImplTest.java

```diff
@@ -59,17 +56,17 @@
 @SpringJUnitConfig
 public class RdrExportServiceImplTest {
   @Autowired private RdrExportService rdrExportService;
+  @Autowired private RdrMapper rdrMapper;
 
-  @MockBean private ApiClient mockApiClient;
-  @MockBean private RdrApi mockRdrApi;
-  @MockBean private RdrExportDao rdrExportDao;
-  @MockBean private UserDao mockUserDao;
-  @MockBean private WorkspaceDao mockWorkspaceDao;
-  @MockBean private WorkspaceService mockWorkspaceService;
-  @MockBean private InstitutionService institutionService;
-  @MockBean private VerifiedInstitutionalAffiliationDao verifiedInstitutionalAffiliationDao;
-  @MockBean private RdrMapper rdrMapper;
-  private DbWorkspace mockWorkspace, mockDeletedWorkspace, mockCreatorWorkspace;
+  @Autowired private ApiClient mockApiClient;
+  @Autowired private RdrApi mockRdrApi;
+  @Autowired private RdrExportDao rdrExportDao;
+  @Autowired private UserDao mockUserDao;
+  @Autowired private WorkspaceDao mockWorkspaceDao;
+  @Autowired private WorkspaceService mockWorkspaceService;
+  @Autowired private VerifiedInstitutionalAffiliationDao verifiedInstitutionalAffiliationDao;
+
+  private DbWorkspace workspace, deletedWorkspace, creatorWorkspace;
```

Comment:
Hmm. I would say multiple declarations on one line is pretty atypical for Java, though I also don't feel very strongly on this one. Since the warning is already configured I probably won't remove it - done.